lib/htmr-transformers.tsx

```diff
@@ -82,6 +84,27 @@ const strong = (node: JSX.IntrinsicElements["strong"]) => {
   return <strong className="font-bold text-gray-900">{children}</strong>;
 };
 
+const img = (node: JSX.IntrinsicElements["img"]) => {
+  const { alt, src } = node;
+
+  const srcPath = getUniquePath(src);
+
+  if (!srcPath) return <br />; // return an element to avoid type-checking error
+
+  return (
+    <Image
+      alt={alt ?? ""}
+      blurDataURL={grayBlur}
+      height={360}
+      layout="responsive"
+      loading="lazy"
+      placeholder="blur"
+      src={srcPath}
+      width={720}
+    />
+  );
+};
+
```

Comment:
Anyone has hints/example on how to test this part?

Reply:
Or you can add testing on another part just to increase the coverage

Reply:
Galaxy brain moment: https://github.com/kawalcovid19/wargabantuwarga.com/pull/556/commits/02243fb6cc9afdd2f35f03c6bc8d3154eed4b1ab

Reply:
Turned out increasing coverage _elsewhere_ did not work @mazipan 😬 , codecov checks new code in this PR separately.

Reply:
After exploring a bit more, I decided I'm going to make a `SimpleImage` UI component with its own test instead of using either Nextjs' `Image` or plain `img`. I will have time to finish it this evening and will update the details then. Marking as resolved for now.